shuup/core/models/_categories.py

```diff
@@ -136,10 +136,10 @@ def is_visible(self, customer):
         return True
 
     @staticmethod
-    def _get_slug_name(self):
+    def _get_slug_name(self, translation):
```

Comment:
does this work? `self.safe_translation_getter('title', language_code=language_code)`
If you pass the `language_code` to this function then `generate_multilanguage_slugs` could be made much simpler